app/forms/publish/course_rollover_form.rb

```diff
@@ -0,0 +1,21 @@
+module Publish
+  class CourseRolloverForm
+    include ActiveModel::Model
+
+    attr_accessor :course
+
+    def initialize(course)
+      @course = course
+    end
+
+    validate :course_is_rollable?
+
+  private
+
+    def course_is_rollable?
+      return if %i[draft empty rolled_over].include? course.content_status
+
+      errors.add(:course_is_rollable?, message: I18n.t("activemodel.errors.models.publish/course_rollover_form.course_is_not_rollable"))
```

Comment:
Not sure that is correct, if we do that we get
```
pry(#<RSpec::ExampleGroups::PublishCourseRolloverForm::WithdrawnCourse>)> subject.errors
=> #<ActiveModel::Errors [#<ActiveModel::Error attribute=course_is_not_rollable, type=invalid, options={}>]>
```
whereas as it is we get
```
=> #<ActiveModel::Errors [#<ActiveModel::Error attribute=course_is_rollable?, type=invalid, options={:message=>"Course must have draft, empty or rolled over status."}>]>
```
I think it is because we are using activemodel instead of activerecord

Reply:
Then the en.yml should be something like

```
activemodel:
    errors:
      models:
        publish/course_rollover_form:
          attributes:
            course_is_not_rollable:
              invalid: "Error message"
```

Reply:
Done